scripts/clean.js

```diff
@@ -26,7 +26,6 @@ const __dirname = dirname(fileURLToPath(import.meta.url));
 const root = join(__dirname, '..');
 
 // remove npm install/build artifacts
-rmSync(join(root, 'node_modules'), { recursive: true, force: true });
```

Comment:
Maybe this behavior is desirable in some cases, but I think it should be an option, because otherwise our `clean` script makes running any other scripts impossible, and we have to `npm install` afterwards.